spiketools/plts/trials.py

```diff
@@ -66,9 +66,9 @@ def plot_rasters(data, vline=None, colors=None, shade=None, show_axis=False, ax=
 
 @savefig
 @set_plt_kwargs
-def plot_firing_rates(x_vals, y_vals, average=None, shade=None, labels=None,
-                      stats=None, sig_level=0.05, ax=None, **plt_kwargs):
-    """Plot continuous spike firing rates.
+def plot_rate_by_time(x_vals, y_vals, average=None, shade=None, labels=None,
```

Comment:
This is minor, but I kinda like the `frs` instead of `firing_rates`, it's just shorter, and also it seems to me that something like `plot_frs_by_time` sounds more intuitive than `plot_rate_by_time`.

Reply:
Hmm, interesting. I had trouble naming this one. I've been trying to avoid acronyms when I can, so that's sorta why I went with `rate(s)`. I could see it both ways I think. I'll think about this a bit more and check back as I continue the edits.